test-app/gradle.properties

```diff
@@ -0,0 +1,16 @@
+mlHost=localhost
+# Define these in gradle-local.properties
+mlUsername=
+mlPassword=
+
+mlStagingForestsPerHost=1
+mlFinalForestsPerHost=1
+mlJobForestsPerHost=1
+
+# For the Confluent Connector Archive
+componentOwner=marklogic
```

Comment:
These seem like they should be in the top-level gradle.properties, not part of the test-app.